language/classify_text/classify_text_tutorial.py

```diff
@@ -90,7 +87,7 @@ def index(path, index_file):
             print('Failed to process {}'.format(file_path))
 
     with io.open(index_file, 'w', encoding='utf-8') as f:
-        f.write(json.dumps(result))
+        f.write(unicode(json.dumps(result)))
```

Comment:
unicode doesn't exist on python 3, what you really want here is:

```
json.dump(result, f)
```

Reply:
python 2.7's json.dump seems to use f.write and `json.dump(result, f)` still returns the following error:

```TypeError: write() argument 1 must be unicode, not str```